src/model.h

```diff
@@ -60,7 +60,7 @@ class Model
   void
   recursiveTextureLoad(const struct aiScene *sc, const struct aiNode* nd);
   void
-  recursive_render(const struct aiScene *sc, const struct aiNode* nd) const;
+  recursive_render(const struct aiScene *sc, const struct aiNode* nd, const int j) const;
```

Comment:
cv qualifiers for integral types are useless in definition. Please remove the const (for integral types, it's only useful for definitions, not declarations).